packages/vehicles/src/index.story.js

```diff
@@ -0,0 +1,190 @@
+import React from "react";
+
+import { withA11y } from "@storybook/addon-a11y";
+import { action } from "@storybook/addon-actions";
+import { withInfo } from "@storybook/addon-info";
+import { storiesOf } from "@storybook/react";
+import {
+  withKnobs,
+  text,
+  color,
+  boolean,
+  select
+} from "@storybook/addon-knobs";
+
+import "../__mocks__/map.css";
+import BaseMap from "@opentripplanner/base-map";
+
+import Vehicles from ".";
+import VehicleLayer from "./VehicleLayer";
+import VehicleGeometry from "./VehicleGeometry";
+import * as utils from "./utils";
+
+// mock data
+const geojson = require("../__mocks__/lineGeom100.json");
+const v1 = require("../__mocks__/all1.json");
+const v2 = require("../__mocks__/all2.json");
+const v3 = require("../__mocks__/all3.json");
+const line = require("../__mocks__/line100.json");
+const all = require("../__mocks__/all.json");
+
+const v = [v1, v2, v3];
+
+const portland = [45.523, -122.671];
+const setTracked = action("setTracked");
+
+const trips = {
+  9563136: "9563136",
+  9563137: "9563137",
+  9563138: "9563138",
+  9562510: "9562510",
+  9562512: "9562512",
+  9562514: "9562514"
+};
+
+function allExample() {
+  const data = utils.reverseGeojsonPointsInGeom(geojson);
+  const pattern = {
+    id: "1",
+    data
+  };
+  const tracked = utils.findVehicleById(
+    line,
+    select("Tracked Vehicle", trips, "9562512")
+  );
+
+  const retVal = (
+    <BaseMap center={portland}>
+      <VehicleLayer
+        name="Real-Time Buses and Trains"
+        vehicles={all}
+        setTracked={setTracked}
+        trackedVehicle={tracked}
+        color={color("tracked color:", "#d54a40")}
+        visible
+      />
+      <VehicleGeometry
+        trackedVehicle={tracked}
+        pattern={pattern}
+        color={color("tracked color:", "#d54a40")}
+        visible
+      />
+    </BaseMap>
+  );
+  return retVal;
```

Comment:
The contents of `retVal` could be returned directly without need for a variable definition.

Reply:
fixed